uaclient/status.py

```diff
@@ -145,6 +145,9 @@ class UserFacingStatus(enum.Enum):
 MESSAGE_UNENTITLED_TMPL = """\
 This subscription is not entitled to {title}.
 For more information see: https://ubuntu.com/advantage"""
+MESSAGE_UNSUPPORTED_PREMIUM = """\
```

Comment:
This appears to be unused?

Reply:
Only due to an oversight for not pulling in the identity module. I have since pulled the identity module into this branch and resolved to source a common MESSAGE_UNSUPPORTED_PREMIUM_CLOUD_TYPE message which cli also uses.